pkgs/modules/pyright-extended/default.nix

```diff
@@ -0,0 +1,50 @@
+{ pkgs, lib, ... }:
+let
+  version = "1.1.317";
+
+  pyright-extended = pkgs.stdenvNoCC.mkDerivation rec {
+    pname = "pyright-extended";
+    inherit version;
+
+    src = pkgs.fetchurl {
+      url = "https://registry.npmjs.org/@replit/pyright-extended/-/pyright-extended-${version}.tgz";
+      hash = "sha256-I/NAp7EjX0a4jzXWczUyOPC8ZFYZibPQLH5vHb9ixZU=";
+    };
+
+    binPath = lib.makeBinPath [
+      pkgs.ruff
```

Comment:
Is this pinnable? Unclear what version this is

Reply:
It's pinned by the flake lockfile to 0.0.269:

```
$ cat /nix/store/93x7m589i3rsjr86fh501hm4hcc0cfmd-pyright-extended-1.1.317/bin/langserver.index.js
#! /nix/store/7q1b1bsmxi91zci6g8714rcljl620y7f-bash-5.2-p15/bin/bash -e
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/0g3sc6fhnhsxx8d837kyj06qcbjind1b-nodejs-18.16.1/bin'':'/':'}
PATH='/nix/store/0g3sc6fhnhsxx8d837kyj06qcbjind1b-nodejs-18.16.1/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/abqs90gi153i6fb0askc0g0gc1miflnd-python3.10-yapf-0.32.0/bin'':'/':'}
PATH='/nix/store/abqs90gi153i6fb0askc0g0gc1miflnd-python3.10-yapf-0.32.0/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/d7i6n067c1ww3ncl5z7rfy5jb3maj9gw-ruff-0.0.269/bin'':'/':'}
PATH='/nix/store/d7i6n067c1ww3ncl5z7rfy5jb3maj9gw-ruff-0.0.269/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
exec "/nix/store/93x7m589i3rsjr86fh501hm4hcc0cfmd-pyright-extended-1.1.317/lib/langserver.index.js"  "$@"
```
If we bumped the flake input, it would potentially change. Do we need an assertion that it is a specific version?